beginner/apps/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![no_std]
+
+use panic_probe as _;
+
+// same panicking *behavior* as `panic-probe` but doesn't print a panic message
+// this prevents the panic message being printed *twice* when `defmt::panic` is invoked
+#[defmt::panic_handler]
+fn panic() -> ! {
+    cortex_m::asm::udf()
+}
+
+/// Terminates the application and makes `probe-run` exit with exit-code = 0
+pub fn exit() -> ! {
```

Comment:
same question I made `advanced/firmware`: is this function used in application code?

Reply:
it's not, removed :)